nucleus/admin/cli/src/main/java/com/sun/enterprise/admin/cli/CLICommand.java

```diff
@@ -1440,12 +1440,24 @@ protected void buildTerminal() {
     
     protected void buildLineReader() {
         if (lineReader == null) {
-            lineReader = LineReaderBuilder.builder()
+            lineReader = newLineReaderBuilder()
                     .terminal(terminal)
                     .build();
         }
     }
     
+    protected LineReaderBuilder newLineReaderBuilder() {
+    	// In community this should be disabled by default
+    	boolean disabled = true;
+    	if(System.getenv("DISABLE_EVENT_EXPANSION") != null) {
+    		disabled = Boolean.valueOf(System.getenv("DISABLE_EVENT_EXPANSION"));
+    	}
+        return LineReaderBuilder.builder()
```

Comment:
indents need fixing, they are misaligned

Reply:
I believe that is just github, if you view the file instead of the snippet, it is aligned.

Reply:
Are there any tabs here by any chance? If yes, please reformat the code w/space and 4 spaces per ident.

Thank you!

Reply:
Should be fixed